spec/rails/auth/x509/middleware_spec.rb

```diff
@@ -49,10 +49,11 @@
 
       let(:java_cert) do
         ruby_cert = OpenSSL::X509::Certificate.new(valid_cert_pem)
-        Java::SunSecurityX509::X509CertImpl.new(ruby_cert.to_der.to_java_bytes)
+        java_cert_klass = java.security.cert::CertificateFactory.getInstance("X.509")
```

Comment:
@nerdrew Do we have a preference on the Ruby class names vs the Java ones?

Reply:
I didn't know how to invoke the instance using the Ruby class names, initially, but I can swap this for `Java::JavaSecurityCert::CertificateFactory.getInstance("X.509")`, if that's preferrable.

Reply:
fixed it